cmds/http-gateway/main.go

```diff
@@ -176,16 +179,37 @@ func myHTTPError(ctx context.Context, mux *runtime.ServeMux, marshaler runtime.M
 	}
 	w.Header().Set("Content-Type", contentType)
 
-	body := &errorBody{
-		Error:   s.Message(),
-		Message: s.Message(),
-		Code:    int32(s.Code()),
-		Details: s.Proto().GetDetails(),
-	}
+	var buf []byte
+	var merr error
+	if s.Code() == errors.MissingOVNs {
+		// Handle special return schema for missing OVNs
+		if len(s.Details()) < 1 {
+			grpclog.Infof("Missing Details from Status")
+			merr = errors.Internal("Missing Details from Status")
+		} else {
+			body, ok := s.Details()[0].(*scdpb.AirspaceConflictResponse)
+			if ok {
+				buf, merr = marshaler.Marshal(body)
+			} else {
+				grpclog.Infof("Cast to *scdpb.AirspaceConflictResponse failed from type %s", reflect.TypeOf(s.Details()[0]))
+				merr = errors.Internal("Unable to cast s.Details()[0] to *scdpb.AirspaceConflictResponse")
+			}
+		}
```

Comment:
alternatively, we could just do another check on the path here: https://github.com/interuss/dss/blob/master/cmds/http-gateway/main.go#L81

and write our own func that calls out to the gprc-backend, which would allow us to keep the backend simple

Reply:
The last section in the first link looks like custom handling of the entire method.  I don't think we want to implement an entire custom handler for this entire method; we just want to augment returned content upon a certain error.  Even if we did write our own function to call out to the grpc-backend, the grpc-backend would still need to return different content in different circumstances.  We would have to write a new proto to handle that also -- it seems like this way is much easier.

I'll add a comment indicating when the error handler function is hit, and what is available at that time.

Reply:
yup, that was me going of grpc sending both status with a message